topicgenerator/README.md

```diff
@@ -24,30 +25,41 @@ openai-api-key: <OpenAI API key>
         - `https://qa.dockstore.org/api`
         - `https://staging.dockstore.org/api`
         - `https://dockstore.org/api`
-- `openai-api-key`: The OpenAI API key required for using the OpenAI APIs. See https://platform.openai.com/docs/api-reference/authentication for more details.
+- `token`: The Dockstore token of an admin or curator. This token is used to upload topics to the webservice.
+- `openai-api-key`: The OpenAI API key required for using the OpenAI APIs. See https://platform.openai.com/docs/api-reference/authentication for more details. This is used to generate topics.
 
 ## Running the program
 
 ```
 Usage: <main class> [options] [command] [command options]
   Options:
+    -c, --config
+      The config file path.
+      Default: ./topic-generator.config
     --help
       Prints help for topicgenerator
   Commands:
     generate-topics      Generate topics for public Dockstore entries using 
-            the gpt-3.5-turbo-16k AI model
+            the gpt-3.5-turbo AI model. Use the upload-topics command to 
```

Comment:
Presumably once you have the `--entries` file, the topics could come from anywhere (i.e. may as well leave out the version for when we upgrade to GPT5 or a competitor)